### ISSUE TITLE
Question:

### ISSUE THREAD

**Issue by Walli2003** [SuperFlyTV/SuperConductor#210]
is it somehow possible to pair streamdeck with buttons that start or pause the whole Group. and at the same time when another one is turned on so that the previous one is paused? Thanks :D

**Comment by nytamin** [comment]
> is it somehow possible to pair streamdeck with buttons that start or pause the whole Group.

Not exactly, however you _could_ assign a "Button Area" (with just one button in it) to a group, then that button would play the first item in the group.

> when another one is turned on so that the previous one is paused?

Not at the moment, no. This is something I've been thinking about adding. (In Sofie the feature is called "exclusivity group".)
